python/ray/data/read_api.py

```diff
@@ -674,6 +679,9 @@ def read_parquet(
         include_paths: If ``True``, include the path to each file. File paths are
             stored in the ``'path'`` column.
         file_extensions: A list of file extensions to filter files by.
+        concurrency: The maximum number of Ray tasks to run concurrently. Set this
```

Comment:
maybe also mention that by default Ray decide concurrency dynamically based on available resources. and this parameter allows users to manually limit the max concurrency for each op.